schema/cql/cql.go

```diff
@@ -55,15 +55,30 @@ func typeMap(t dosa.Type) string {
 	return "unknown"
 }
 
+func selectFieldsInCreatingView(columns []string) string {
+	if len(columns) == 0 {
+		return "*"
+	}
+	fields := ""
+	for colnum, col := range columns {
```

Comment:
You can use `strings.Join` here, https://golang.org/pkg/strings/#Join

Reply:
Cool! strings magic!